webhooks/postgres/init.go

```diff
@@ -46,10 +46,9 @@ func migrateDB(db *sqlx.DB) error {
 				Id: "webhooks_1",
 				Up: []string{
 					`CREATE TABLE IF NOT EXISTS webhooks (
-						thing_id   UUID NOT NULL,
-						name       VARCHAR(1024) NOT NULL,
-						format     VARCHAR(60) NOT NULL,
-						url        VARCHAR(254) NOT NULL,
+						thing_id   		UUID NOT NULL,
+						name       		VARCHAR(1024) NOT NULL,
+						url        		VARCHAR(254) NOT NULL,
```

Comment:
Copy the coding style from here: https://github.com/MainfluxLabs/mainflux/blob/master/things/postgres/init.go#L51